app/routes/all-questions/index.tsx

```diff
@@ -7,7 +7,22 @@ import { getContracts } from "~/services/contracts.server";
 
 export async function loader() {
   const network = process.env.NETWORK || "localhost";
-  const { bountyQuestionJson, questionAPIJson, questionStateController } = getContracts({network: network});
+  const contractData = getContracts({ network: network });
```

Comment:
you can move the mappings below to the getContracts method simplifying this to:

```
const network = process.env.NETWORK || "localhost";
return getContracts({ network: network });
```

Reply:
True but you still have to filter out which contracts sent to front end so it would be more like
```
  const { bountyQuestionJson, questionAPIJson, questionStateController } = getContracts({ network: network });

  return {
    bountyQuestionJson,
    questionAPIJson,
    questionStateController,
    network: network,
  };
  ```